README.md

```diff
@@ -5,23 +5,291 @@
+-  [Working groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md)
+
+### Background
+
+We reorganized the current [helm installation parameters](https://istio.io/docs/reference/config/installation-options/) into two groups:
+
+-  The new [platform level installation API](https://github.com/istio/operator/blob/master/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto), for managing
+k8s settings like resources, auto scaling, pod disruption budgets etc. 
+-  The configuration API that currently uses the
+[helm installation parameters](https://istio.io/docs/reference/config/installation-options/) for backwards
+compatibility. This API is for managing This API the Istio control plane configuration settings.
+
+Some parameters will temporarily exist in both APIs - for example, setting k8s resources currently can be done through
+either API above. However, the Istio community recommends using the first API as it is more consistent, is validated,
+and will naturally follow the graduation process for APIs while the same parameters in the configuration API are planned
+for deprecation.
+
+We currently provide pre-configured helm values sets for different scenarios as configuration 
+[profiles](https://istio.io/docs/setup/kubernetes/additional-setup/config-profiles/), which act as a starting point for
+an Istio install and can be customized by creating customization overlay files or passing parameters when
+calling helm. Similarly, the operator API uses the same profiles (expressed internally through the new API), which can be selected
+as a starting point for the installation. For comparison, the following example shows the command needed to install
+Istio using the SDS configuration profile using Helm:
+
+```bash
+helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+    --values install/kubernetes/helm/istio/values-istio-sds-auth.yaml | kubectl apply -f -
+```
+
+In the new API, the same profile would be selected through a CustomResource (CR):
+
+```yaml
+# sds-install-cr.yaml
+
+apiVersion: install.istio.io/v1alpha1
+kind: IstioControlPlane
+metadata:
+  name: istio-operator-config
+  namespace: istio-system
+spec:
+  profile: sds
+```
+
+CRs are used when running the operator as a controller in a pod in the cluster. When using operator CLI mode and passing the 
+configuration as a file (see [Select a profile](#Select_a_profile)), only the spec portion is required.
+
+```yaml
+# sds-install.yaml
+
+profile: sds
+```
+
+If you don't specify a configuration profile, Istio is installed using the `default` configuration profile. All 
+profiles listed in istio.io are available compiled in, or `profile:` can point to a local file path to reference a custom
+profile base to use as a starting point for customization. See the [API reference](https://github.com/istio/operator/blob/master/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto)
+for details.
+
+## Developer quick start
```

Comment:
although I disagree with the choice, I think project wide, the docs WG has made a decision that developer material belongs in the consolidated wiki.  @rcaballeromx is this accurate?